### ISSUE TITLE
Docker - obiba/opal:4.5

### ISSUE THREAD

**Issue by jonathanmasseurca** [obiba/docker-opal#22]
**Describe the bug**

Opal version 4.5.5

After every restart the settings (only the line) ( org.obiba.real.url ) in opal-config.properties is lost.

Other lines dont have this issue, not sure why the file is overwriting this line.

**To Reproduce**
docker restart "dockerid"

**Expected behavior**
Keep org.obiba.realm.url line and have agate working with opal.

**Comment by ymarcon** [comment]
I cannot reproduce on opal-demo.obiba.org and nothing has changed on that setting probably for years.

**Comment by ymarcon** [comment]
Try to inquire why the first_run.sh script is run after the restart (not supposed to happen if the file `/opt/opal/bin/first_run.sh.done` exists). This is the code where agate url is set or disabled:

https://github.com/obiba/docker-opal/blob/master/bin/start.sh#L46-L69

**Comment by jonathanmasseurca** [comment]
Thank you @ymarcon, you're the best, it was the issue, you definitely saved me alot of time.

I added a definition for the environment variable (in the docker-compose) of opal and it fixed the issue. I think maybe someone manually defined the environement variable in the old version 4.5.4 by running a bash command line and that's why it was working, so when I recreated the container it reseted the environment variable that was not in part of the docker-compose.xml


Is there any reason why if it's not defined, it's erasing the config file? Maybe it's would be better to remove the 3 lines in 
https://github.com/obiba/docker-opal/blob/master/bin/first_run.sh

[  else
    echo "Disabling default Agate setting as AGATE_HOST is not defined..."
    set_property "org.obiba.realm.url" "" "$OPAL_HOME/conf/opal-config.properties"
](https://github.com/obiba/docker-opal/blob/master/bin/start.sh#L66-L68)



Also the docker-compose.xml never show the variables AGATE_HOST and AGATE_PORT, (this is where I added the two missing variables)

https://github.com/obiba/docker-opal/blob/master/docker-compose.yml#L15-L33

Many thanks again
